src/AddressSplitter.php

```diff
@@ -31,7 +31,7 @@ public static function splitAddress($address)
             (?:No\.\s*)?
                 (?P<A_House_number_match>
                      (?P<A_House_number_base>
-                        \pN+
+                        \pN+(\s+\d\/\d)?
```

Comment:
What about "2 13/28"? This housenumber will not match the expression. In my opinion the new condition should be like

```
(\s+\d+\/\d+)?
```

Reply:
i thing this is a very very rare and constructed case.

Reply:
Ooookay, i did a bit of a research and found this: https://de.wikipedia.org/wiki/Hausnummer#Hausnummernerg%C3%A4nzungen
So we should change it